.esh/commands/locales-full.py

```
@@ -29,200 +29,200 @@
-            # 存在 CRLF 或尾随 \r，需要写回以统一为 LF
-            pass
-        elif raw.decode("utf-8") == content:
-            return False
-    with path.open("w", encoding="utf-8", newline="\n") as f:
-        f.write(content)
-    return True
+	path.parent.mkdir(parents=True, exist_ok=True)
+	if path.exists():
+		raw = path.read_bytes()
+		if b"\r\n" in raw or raw.endswith(b"\r"):
+			# 存在 CRLF 或尾随 \r，需要写回以统一为 LF
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)